packages/utils/upgrade/src/types.ts

```diff
@@ -43,3 +43,13 @@ export interface Report {
   nochange: number;
   skip: number;
 }
+
```

Comment:
Can we combine this with the one in @strapi/types so that we don't duplicate it? Maybe it could live in @strapi/types/utils and both this and entity service (and document service) can pull in that